### ISSUE TITLE
Unable to get New Relic Alert Policy data source

### ISSUE THREAD

**Issue by alivirya** [newrelic/terraform-provider-newrelic#755]
Getting `Access Denied., null` when trying to import a new relic alert policy in terraform as a data source.

### Terraform Version
0.12.24

### Debug Output
https://gist.github.com/alivirya/5caf3fa102770036f1589f181ca9d64f

Have submitted a query via postman with the query below that works so it doesn't seem like an api key issue:
```
  {
    actor {
        account(id: {accountId}) {
            alerts {
                policy(id: {policyId}) {
                    id
                    name
                    incidentPreference
                    accountId
                    }
            }
        }
    }
}
```

### Expected Behavior
I've created a New Relic Alert Policy data source that imports from an existing new relic alert policy. It should be able to import this information. 

### Actual Behavior
Getting `Access Denied., null`, when looking at the logs and the Graphql output, I tried running the postman query myself and it seems to work so it doesn't seem to be a problem with the api keys. Seems like this might be a problem with how the provider performs the query

### Steps to Reproduce
Create a New Relic data source
```
# data "newrelic_alert_policy" "name" {
#   name = "{existing_policy_name}"
# }
```
`terraform plan`

**Comment by ctrombley** [comment]
Hi @alivirya!

Can you confirm that that `account_id` configured in your [provider block](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/guides/provider_configuration) matches the account ID you are using in your test query?

**Comment by alivirya** [comment]
@ctrombley I've only defined the `account_id` in the provider block. It is not referenced in the data source?

**Comment by ctrombley** [comment]
Resolved by #783 .

**Comment by ctrombley** [comment]
Duplicate of #773
